ExceptionHandle Middleware

YaroslavV1 opened the pull request:
feat - Created base exceptions;
feat - Created supporting model;
feat - Created and initialize ExceptionMiddleware.

YaroslavV1 committed:
feat - Created base exceptions;
feat - Created supporting model;
feat - Created and initialize ExceptionMiddleware.

Vladislawv commented on LifeHelper.Api/Middlewares/ExceptionHandlerMiddleware.cs:
```diff
@@ -0,0 +1,52 @@
+    }
+
+    private async Task ExceptionAsync(Exception exception, HttpContext context)
+    {
```
it`s better to call this method HandleExceptionAsync

Vladislawv commented on LifeHelper.Infrastructure/Exceptions/BadRequestException.cs:
```diff
@@ -0,0 +1,9 @@
+{
+    private const HttpStatusCode StatusCode = HttpStatusCode.BadRequest;
+    protected BadRequestException(string message) : base(StatusCode, message) {}
+}
```
it`s good to take one empty line between fields and constructor

Vladislawv commented on LifeHelper.Infrastructure/Exceptions/NotFoundException.cs:
```diff
@@ -0,0 +1,9 @@
+{
+    private const HttpStatusCode StatusCode = HttpStatusCode.NotFound;
+    protected NotFoundException( string message) : base(StatusCode, message) {}
+}
```
also need empty line between 7 and 8

YaroslavV1 committed:
fix - Fix custom exceptions view;
fix - Rename private method in ExceptionHandleMiddleware;
fix - Fix bug in the appsettings.json.
